cli/src/config/misc.toml

```diff
@@ -47,3 +47,7 @@ wrapping = "anywhere"
 max-new-file-size = "1MiB"
 auto-track = "all()"
 auto-update-stale = false
+
+# TODO: https://github.com/jj-vcs/jj/issues/3419 - Remove when the fully deprecated.
```

Comment:
nit: delete "the", i guess (or just mark this resolved if you prefer - it doesn't really matter)

Reply:
Ah, how aggravating. Fixed now.